snowflake-resource-optimization/setup-guides/set-up-snowflake.md

```diff
@@ -0,0 +1,19 @@
+# Snowflake Resource Optimization Setup - Step 1: Snowflake
+
+## What's Snowflake?
+Snowflake provides cloud-based data storage and analytics services, and if you're here, you're likely already using it! The Snowflake Resource Optimization: Setup & Configuration app will read from Snowflake to populate its queries, and write back to Snowflake as users take actions (e.g. enabling auto-resume for a warehouse or disabling an idle user).
```

Comment:
Suggestion for the second sentence -- we can keep it simple b/c of the fuller context in the main README. (One phrase—"to populate its queries"—actually stuck out as being potentially confusing if you don't immediately make the connection that queries = Retool app Queries.)


"The [Retool app.. whatever we name it :p] is designed to help you manage your Snowflake resources."

Reply:
done